contracts/modules/ProtocolSettings.sol

```diff
@@ -403,13 +403,15 @@ contract ProtocolSettings is State, ProtocolTokenUser, ProtocolSettingsEvents, M
 			} else {
 				if (tokens[i] == address(wrbtcToken)) {
 					amountConvertedToWRBTC = tempAmount;
+
+					IERC20(address(wrbtcToken)).safeTransfer(receiver, amountConvertedToWRBTC);
 				} else {
 					IERC20(tokens[i]).approve(protocolAddress, tempAmount);
 
 					(amountConvertedToWRBTC, ) = ProtocolSwapExternalInterface(protocolAddress).swapExternal(
 						tokens[i], // source token address
 						address(wrbtcToken), // dest token address
-						feesController, // set feeSharingProxy as receiver
+						feesController, // set protocol as receiver
```

Comment:
Receiver is still rge fee sharing proxy, isn't it? Why did you chenge the comment?

Apart from that all looks good to me.